cg/apps/loqus.py

```diff
@@ -27,8 +27,8 @@ def __init__(self, config: dict, analysis_type: str = "wgs"):
         self.loqusdb_binary = config["loqusdb"]["binary_path"]
 
         if self.analysis_type == "wes":
-            self.loqusdb_config = config["loqusdb-wes"]["config_path"]
-            self.loqusdb_binary = config["loqusdb-wes"]["binary_path"]
+            self.loqusdb_config = config["loqusdb_wes"]["config_path"]
```

Comment:
I can still see this variable in kebab-case in servers repo

Reply:
what is kebab-case??

Reply:
I'm not sure what you mean but in the `cg_config.yaml` it will look like `loqusdb-wes` and internally that will get mapped (via alias in pydantic) to `loqusdb_wes`. So when converted back do a dict, which is being done in the CGConfig.

Reply:
`kebab-case` is in kebab-case! It has a dash instead of an underscore as a separator, resembling a kebab skewer.